packages/tupaia-web/src/views/ProjectsModal.tsx

```diff
@@ -132,19 +130,25 @@ export const ProjectsModal = () => {
         ) : (
           <ProjectsGrid>
             <ProjectCardList
-              projects={projects}
+              projects={data?.projects ?? []}
               actions={{
                 [PROJECT_ACCESS_TYPES.ALLOWED]: ({
                   project: { code, homeEntityCode, dashboardGroupName },
+                }: {
+                  project: SingleProject;
                 }) => (
                   <ProjectAllowedLink
                     url={`/${code}/${homeEntityCode}${
-                      dashboardGroupName ? `/${dashboardGroupName}` : ''
+                      dashboardGroupName ? `/${encodeURIComponent(dashboardGroupName)}` : ''
```

Comment:
good catch. Is this going to be an issue in other places too? Such as MultiProjectLandingPage and SingleProjectLandingPage and useNavigateToEntity?